imageregistry/v1/types.go

```diff
@@ -207,17 +207,7 @@ type ImageRegistryConfigStorageS3 struct {
 	// Optional, defaults based on the Region that is provided.
 	// +optional
 	RegionEndpoint string `json:"regionEndpoint,omitempty"`
-	// chunkSizeMiB defines the size of the multipart upload chunks of the S3 API.
-	// The S3 API requires multipart upload chunks to be at least 5MiB.
-	// When omitted, this means no opinion and the platform is left to choose a reasonable default, which is subject to change over time.
-	// The current default value is 10 MiB.
-	// The value is an integer number of MiB.
-	// The minimum value is 5 and the maximum value is 5120 (5 GiB).
-	// +kubebuilder:validation:Minimum=5
-	// +kubebuilder:validation:Maximum=5120
-	// +openshift:enable:FeatureGate=ChunkSizeMiB
-	// +optional
-	ChunkSizeMiB int32 `json:"chunkSizeMiB,omitempty"`
```

Comment:
This field is unrelated to the feature flag. If we really want to remove it we'd need to go through a deprecation process, but I suspect that wasn't your intention here.